server/scripts/availability_dump.ts

```diff
@@ -112,20 +112,24 @@ async function getAllBucketObjects(options) {
  * performs *terribly* and is often worse than no chunking at all.
  */
 class BufferedStream extends stream.Transform {
+  #buffer: Buffer = null;
+  #size = 0;
+  #offset = 0;
```

Comment:
Fancy-pants private properties! Don’t know why I wasn’t using these before; we don’t need TypeScript for them in the current version of Node.js.